src/laplace.jl

```diff
@@ -0,0 +1,16 @@
+struct Laplacian{T,D} <: LazyQuasiMatrix{T}
+    axis::Inclusion{T,D}
+end
+
+Laplacian{T}(axis::Inclusion{<:Any,D}) where {T,D} = Laplacian{T,D}(axis)
+# Laplacian{T}(domain) where T = Laplacian{T}(Inclusion(domain))
+# Laplacian(axis) = Laplacian{eltype(axis)}(axis)
+
+axes(D::Laplacian) = (D.axis, D.axis)
+==(a::Laplacian, b::Laplacian) = a.axis == b.axis
+copy(D::Laplacian) = Laplacian(copy(D.axis))
+
+@simplify function *(Δ::Laplacian, P::AbstractSphericalHarmonic)
```

Comment:
Should there be a check that "the right Laplacian" is being applied? Right now, this doesn't care whether the Laplacian was actually generated for a SphericalHarmonic or whatever other basis in a different OP package. Maybe that makes sense design wise in that the operator just knows what to do but then it's a bit weird that we're generating it via `Laplacian(axes(SphericalHarmonic(),1))`. I guess that's just a question for future consistency.

Reply:
I believe the ` @simplify ` takes care of that